inventory/byo/hosts.origin.example

```diff
@@ -613,7 +613,10 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # WORKAROUND : If you must use an overlapping subnet, you can configure a non conflicting
 # docker0 CIDR range by adding '--bip=192.168.2.1/24' to DOCKER_NETWORK_OPTIONS
 # environment variable located in /etc/sysconfig/docker-network.
-# When upgrading these must be specificed!
+# When upgrading or scaling up the following must match whats in your master config!
```

Comment:
It's not clear that these are now mandatory and install will fail without them.

Reply:
I'll clarify.